### PR TITLE
[cluster_test] Fix 3-region simulation

### PR DESCRIPTION
**Pull request by andll**
Root cause is having `sudo` in network effects cmd line.

Relied on land blocking push to catch the issue, but it does not test 3-region simulation, so the problem slipped in master

There was also a typo in `wait_job_completion` so failed job was treated as `still in progress`

### CODE REVIEW

File: testsuite/cluster-test/src/main.rs [hunk 1]
```diff
@@ -763,7 +763,7 @@ impl ClusterTestRunner {
 
     async fn wait_until_all_healthy(&mut self) -> Result<()> {
         info!("Waiting for all validators to be healthy");
-        let wait_deadline = Instant::now() + Duration::from_secs(20 * 60);
+        let wait_deadline = Instant::now() + Duration::from_secs(5 * 60);
```
Comment (snakech) [inline testsuite/cluster-test/src/main.rs:766]: just concern the shorten duration is enough for node recovery?

Comment (andll) [reply testsuite/cluster-test/src/main.rs:766]: Yeah, from all practical purposes recovery happens much faster than 5 minutes.

**Comment by andll** [comment]
@bors-libra r=snakech p=100
Fixes nightly build

**Pull request closed without merging**